assets/styles/layouts/partials/sections/event-highlights.css

```diff
@@ -35,3 +35,8 @@
     justify-content: center;
     flex-wrap: wrap;
 }
+
+.event-highlights-section__description--paragraph {
+    color: var(--color-secondary-darker);
+    line-height: 1.5rem;
```

Comment:
Is the line height modifier specific or the general style of a paragraph?

Reply:
renamed